### PR TITLE
Fix go-get for krew-manifest-validator

### PR DESCRIPTION
**Pull request by ahmetb**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ before_install:
     )
 
   # TODO(ahmetb) pin this to a version via a variable
-  - env GOBIN=$HOME/bin go get github.com/GoogleContainerTools/krew/cmd/validate-krew-manifest
+  - env GOBIN=$HOME/bin go get github.com/kubernetes-sigs/krew/cmd/validate-krew-manifest
```
Comment (corneliusweig) [inline .travis.yml]: @.soltysh pointed out in krew, that it should actually be `sigs.k8s.io/krew` instead of `github.com/kubernetes-sigs/krew`.

**Comment by ahmetb** [comment]
/close
fixed in another PR (minus the krew.yaml changes)

**Pull request closed without merging**